sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/BatchTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleCounter;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+import io.opentelemetry.api.metrics.DoubleUpDownCounter;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.testing.InMemoryMetricReader;
+import java.time.Duration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BatchTest {
+
+  private final InMemoryMetricReader reader = InMemoryMetricReader.create();
+
+  @BeforeEach
+  void setup() {
+    GlobalOpenTelemetry.resetForTest();
+    GlobalOpenTelemetry.set(
+        OpenTelemetrySdk.builder()
+            .setMeterProvider(
+                SdkMeterProvider.builder()
+                    .setMinimumCollectionInterval(Duration.ZERO)
+                    .registerMetricReader(reader)
+                    .build())
+            .build());
+  }
+
+  @AfterEach
+  void cleanup() {
+    GlobalOpenTelemetry.resetForTest();
+  }
+
+  @Test
+  void batch() {
+    Meter meter = GlobalOpenTelemetry.get().getMeterProvider().get("meter");
```

Comment:
curious why you would use the global for this. Why not just construct an instance, rather than rely on the global state and resetting it each time?